frontend/src/views/SQR/Form.vue

```diff
@@ -197,6 +197,7 @@ export default class Form extends Vue {
   showValidationSummary = false;
   getUserClaims!: any;
   rules = [required];
+  branch: SubmissionsBranch | "" = "";
```

Comment:
The data has to have a value to be reactive and I did not want to set a default branch. I have changed it to `null` instead of `""` because I think that makes more sense.

Reply:
OK.